README.md

```diff
@@ -45,6 +45,18 @@ Repository for TDR transfer code
 
 * run  `npm install` then `npm run build`
 
+### Generated GraphQL classes
+There is a separate repository which contains the generated case classes needed to query the consignment API. 
+These classes will be needed by more than one project which is why they are in a separate project.
+If you need to add a new query:
+
+* Run `git clone https://github.com/nationalarchives/tdr-generated-graphql.git`
```

Comment:
Could you link to this from the docs, add a licence, etc?

Reply:
I've added the licence and linked to the main documentation repo.

Reply:
Cheers. :+1: 

BTW, the Jenkins user is a member of the TDR team, so I don't think it needs to be added separately. Or is there are a reason to give it its own write access?

Reply:
Chatted about this in person - we're going to just use the TDR teams to give access.